src/main/java/org/example/constant/UrlPatterns.java

```diff
@@ -0,0 +1,31 @@
+package org.example.constant;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum UrlPatterns {
+    HTML("/html"), JSON("/json"), HELLO("/hello");
+
+    private final String url;
+
+    private static final Map<String, UrlPatterns> URL_PATTERN_STRING_MAP = new HashMap<>();
+
+    static {
+        URL_PATTERN_STRING_MAP.put(HELLO.getUrl(), HELLO);
+        URL_PATTERN_STRING_MAP.put(JSON.getUrl(), JSON);
+        URL_PATTERN_STRING_MAP.put(HTML.getUrl(), HTML);
+    }
+
+    UrlPatterns(String url) {
+        this.url = url;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    // Method for get UrlPattern from the line with Urls
+    public static UrlPatterns getBySymbol(String url) {
```

Comment:
You receive the url with the prefix '/DevelopersApi'.
You should remove this prefix

Reply:
fix